### DISCUSSION TITLE
Let administrators create custom MarkDown Pages

### DISCUSSION THREAD

**Discussion by BelieveInBunny** [torrust/torrust-index-archive#94] [Feature requests]
Allowing users to create pages using markdown

**Comment by mickvandijke** [comment]
This is a great and simple idea! I'll add it to the upcoming release

**Comment by BelieveInBunny** [comment]
Just an update, I am working on this feature and will have it ready in 2-3 days

**Comment by da2ce7** [comment]
- https://github.com/torrust/torrust-index-frontend/discussions/291
